tests/Agent/IntegrationTests/UnboundedApplications/MongoDB2_6Application/MongoDB2_6Application.csproj

```diff
@@ -104,11 +101,10 @@
       <Version>4.3.0</Version>
     </PackageReference>
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation">
-      <Version>4.0.0</Version>
+      <Version>4.3.0</Version>
```

Comment:
Why the change in version here?

Reply:
One class of build warnings was about package versions being downgraded.  Making this change eliminated the warning, and the tests still build and pass.